xdocs/usermanual/component_reference.xml

```diff
@@ -2957,6 +2957,9 @@ may download images, style-sheets, etc. referenced by the HTML code.
 <tr><td><code>JSON</code></td>
 <td>The <i>JSON view</i> will show the response in tree style (also handles JSON embedded in JavaScript).
 <br/></td></tr>
+<tr><td><code>JSON PATH TESTER</code></td>
```

Comment:
`JSON Path Tester`? JSON is an acronym, but path and tester are not and the regexp tester below is written that way.